src/Line.php

```diff
@@ -220,12 +220,15 @@ public function getAttribute($name)
      *
      * Certain elements needs to prepend values into the next element.
      *
+     * > Since version 1.3.3 the Line argument is required in order to ensure the correct index for the prepend element.
```

Comment:
Shouldn't this use `@since` instead of `> Since`?

I guess `> ...` is used for the automatic documentation. But the rest of the code all uses `@since`, so it would be nice for the consistency.

Reply:
I will add `{@since}` for the new property. Good catch